inits.json

```diff
@@ -54,8 +54,8 @@
             "Orbital Period Uncertainty": 1.3e-07,
             "Published Mid-Transit Time (BJD-UTC)": 2455867.402743,
             "Mid-Transit Time Uncertainty": 4.9e-05,
-            "Ratio of Planet to Stellar Radius (Rp/Rs)": 0.14886235252742716,
-            "Ratio of Planet to Stellar Radius (Rp/Rs) Uncertainty": 0.0005539487393037134,
+            "Ratio of Planet to Stellar Radius (Rp/Rs)": 0,
```

Comment:
Are you sure you want to push this? or was this here just for testing?

Reply:
Nope, weird - I didn't add the inits file to the push. Looks like I accidentally did. Will remove it.

Reply:
Ok, reverted the inits file back to what's on dev.